code/game/objects/effects/spawners/ghostroles.dm

```diff
@@ -129,18 +129,39 @@
 		STAT_BIO = 10,
 		STAT_MEC = 5,
 		STAT_VIG = 45,
+		STAT_COG = 20 //twice as smarter as the shieldie
+	)
+
+/obj/effect/mob_spawn/human/blackshield_ert
+	name = "deployment sleeper"
+	desc = "An sleeper, with an unconscious body inside. The occupant seems to be a Nadezhda Blackshield Specialist"
+	mob_name = "a human"
+	icon = 'icons/obj/Cryogenic2.dmi'
+	icon_state = "sleeper_1"
+	outfit = /decl/hierarchy/outfit/blackshield_ert
+	ghost_role_perks = list(PERK_ASS_OF_CONCRETE, PERK_SMARTLINK, PERK_CODESPEAK, PERK_BLACKSHIELD_CONDITIONING, PERK_TRUE_NAME)
```

Comment:
they should get bolt trainning, not smartlink

Reply:
I actually nixed bolt training as it seemed fairly useless for both of them, where as the smartlink is slightly more handy.